### PR TITLE
Fix applicant cannot choose 1.1 as start date

### PR DESCRIPTION
**Pull request by ngocdh236**
## Description :sparkles:
Fix applicant cannot choose 1.1 as start date

### CODE REVIEW

File: frontend/benefit/applicant/src/components/applications/forms/application/step2/utils/validation.ts [hunk 1]
```diff
@@ -76,7 +75,7 @@ export const getValidationSchema = (t: TFunction): Yup.SchemaOf<Step2> =>
         }),
         test: (value = '') => {
           const date = parseDate(value);
-          return date ? isAfter(date, startOfYear(new Date())) : false;
```
Comment (nenonja) [inline frontend/benefit/applicant/src/components/applications/forms/application/step2/utils/validation.ts:79]: Looks okay, but this would be a great moment to add unit tests for the validation too. Not necessary though. It seems that the test method should be first extracted here to be able to test it?

Comment (ngocdh236) [reply frontend/benefit/applicant/src/components/applications/forms/application/step2/utils/validation.ts:79]: I have moved to util and written unit test for this now

File: frontend/shared/src/utils/__tests__/date.utils.test.ts [hunk 2]
```diff
@@ -39,4 +45,19 @@ describe('dates', () => {
       expect(diffMonthsResult).toBeLessThan(0);
     });
   });
+
+  describe('validateDateIsFromCurrentYearOnwards', () => {
+    it('should return false', () => {
+      const validateDateIsFromCurrentYearOnwardsResult =
+        validateDateIsFromCurrentYearOnwards('31.12.2021');
```
Comment (nenonja) [inline frontend/shared/src/utils/__tests__/date.utils.test.ts:52]: `31.12.${new Date().getFullYear()-1}` could more of an edge value in the future too :)
Also If you want to get path coverage 100% then there could be one more test with invalid string value which should return false.

**Review by nenonja — APPROVED** [review]
One minor comment but approved

File: frontend/shared/src/utils/__tests__/date.utils.test.ts (hunk 2 above)
Comment (ngocdh236) [reply frontend/shared/src/utils/__tests__/date.utils.test.ts]: Thank you, I have updated the test, if it's ok then I will merge
